.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.5
```

Comment:
Another thought here is that if we push towards making Inferno into a gem we'll want it to support multiple versions and this won't make as much sense.  Treating it as an application this does make it more reproducible though.

Reply:
I think that's a bridge we should cross when we come to it. Right now Inferno is an application, and I think we should ensure that the production and development environments use the same ruby version.